src/protocols/rdp/rdp_settings.c

```diff
@@ -1007,6 +1023,16 @@ guac_rdp_settings* guac_rdp_parse_args(guac_user* user,
                 IDX_LOAD_BALANCE_INFO, NULL);
 #endif
 
+    /* Parse clipboard copy disable flag */
+    settings->disable_copy =
+        guac_user_parse_args_boolean(user, GUAC_RDP_CLIENT_ARGS, argv,
+                IDX_DISABLE_COPY, 0);
```

Comment:
Any particular reason, for the RDP flags, that you're parsing as `boolean` but using `0` instead of `false` for the default?  Looking at the other protocols you've used `false` - not sure why this is different?

Reply:
Yep - the FreeRDP library defines its own boolean type which breaks any code using `stdbool.h`. We thus can't use the standard `bool` type in any code that touches FreeRDP.

This is why the `enable_sftp` flag is an `int` in the RDP support:

https://github.com/apache/guacamole-server/blob/802e5b5547dc26f1fa7d0abba91570f87cfefb32/src/protocols/rdp/rdp_settings.h#L353

but is `bool` elsewhere:

https://github.com/apache/guacamole-server/blob/802e5b5547dc26f1fa7d0abba91570f87cfefb32/src/protocols/ssh/settings.h#L161